.github/workflows/miniwdl-test.yml

```
@@ -0,0 +1,145 @@
+            <testcase classname="MiniWDL" name="wdl-1.2-conformance">
+              $(if [ "$failed" -gt 0 ]; then echo "<failure message=\"Failures=$failed, Invalid=$invalid\"/>"; fi)
+            </testcase>
+          </testsuite>
+          EOF
+
+          # Copy list of failures to artifacts
+          cp miniwdl_test/failed_tests.txt miniwdl_tests/artifacts/failed_tests.txt
+
+          # Generate shields.io badge JSON
+          color="green"
+          if [ "$passed" -lt "$total" ]; then color="yellow"; fi
+          if [ "$passed" -eq 0 ]; then color="red"; fi
+          cat <<EOF > miniwdl_tests/artifacts/shields.json
+          {
+            "schemaVersion": 1,
+            "label": "MiniWDL WDL 1.2",
+            "message": "$passed/$total passed",
+            "color": "$color"
+          }
+          EOF
+
+          # Do NOT fail the workflow if any failures or invalid outputs occurred
+          if [ "$failed" -gt 0 ] || [ "$invalid" -gt 0 ]; then
+            echo "❌ One or more conformance tests failed or had invalid outputs."
+            # exit 1   # <-- removed
+          fi
+
+      - name: Upload test artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: miniwdl-test-results
+          path: miniwdl_tests/artifacts/
+
+      - name: Publish badge JSON to current branch
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "github-actions[bot]"
+          BRANCH="${GITHUB_REF_NAME}"
+          RUN_ID="${{ github.run_id }}"
+          REPO="${{ github.repository }}"
+          RUN_URL="https://github.com/$REPO/actions/runs/$RUN_ID"
+          REPO_URL="https://x-access-token:${GITHUB_TOKEN}@github.com/${{ github.repository }}.git"
+
+          git fetch origin $BRANCH
+          git checkout $BRANCH
+
+          mkdir -p shields
+          cp miniwdl_tests/artifacts/shields.json shields/miniwdl_shields.json
+          git add shields/miniwdl_shields.json
+
+          # Update README only if needed
+          sed_pattern="https://raw.githubusercontent.com/openwdl/wdl/.*/shields/miniwdl_shields.json"
+          new_url="https://raw.githubusercontent.com/openwdl/wdl/$BRANCH/shields/miniwdl_shields.json"
+          run_pattern="https://github.com/$REPO/actions/runs/[0-9]+"
+          new_run="https://github.com/$REPO/actions/runs/$RUN_ID"
+          if grep -qE "$badge_pattern" README.md; then
+            git pull
+            sed -E -i "/miniwdl_shields\.json/s|$sed_pattern|$new_url|g" README.md
+            sed -E -i "/miniwdl_shields\.json/s|$run_pattern|$new_run|g" README.md
+            git add README.md
+          fi 
+
+          # Only commit if there are staged changes
+          if ! git diff --cached --quiet; then
+            git commit -m "Update miniwdl shields badge for $BRANCH"
```

```suggestion
            git commit -m "chore: updates Miniwdl shields badge for `$BRANCH`"
```

Same comment as above

---

Yep updating